connect/cli/plugins/commerce/utils.py

```diff
@@ -0,0 +1,340 @@
+import os
+import json
+from datetime import datetime
+import string
+
+from click import ClickException
+from connect.cli.core.terminal import console
+from connect.client import ClientError
+from openpyxl import Workbook
+from openpyxl.styles import Alignment, Font, PatternFill
+from openpyxl.styles.colors import WHITE, Color
+
+from connect.cli.core.utils import validate_output_options
+
+
+def display_streams_table(
+    query_billing_streams,
+    query_pricing_streams,
+    active_account_id,
+):
+    rows = []
+    for origin, query in (
+        ('billing', query_billing_streams),
+        ('pricing', query_pricing_streams),
+    ):
+        for resource in query:
+            rows.append(
+                (
+                    resource['id'],
+                    origin,
+                    resource['name'],
+                    'Computed' if ('sources' in resource and resource['sources']) else 'Simple',
+                    'Inbound' if resource['owner']['id'] == active_account_id else 'Outbound',
```

Comment:
This is not correct. 
* In case of vendors, all streams are Outbound and the owner is the active account.
* In case of distributors, it will be Outbound if the owner is the same as the current active account otherwise Inbound.

Reply:
Agree. But if it is a Vendor, he will be the owner of all the streams so it will work. Am I right?